src/java/org/apache/nutch/parse/ParseUtil.java

```diff
@@ -261,19 +263,14 @@ public void processSitemapParse(String url, WebPage page,
 
           Mark.INJECT_MARK.putMark(newRow, InjectType.SITEMAP_INJECT.getTypeString());
```

Comment:
Isn't the sitemap inject marker supposed to make pages AS sitemaps? Why is it being applied to links that were found in sitemaps?